### ISSUE TITLE
Support Processing Group runtime state definition (and a CLI override)

### ISSUE THREAD

**Issue by hijakk** [aperepel/nifi-api-deploy#15]
Halted services starting themselves after receiving an update is not desirable behavior in our environment. Would it be possible to include a command line flag to keep the process groups from starting before we explicitly tell them to through the UI or REST API?

**Comment by aperepel** [comment]
As an alternative, would supporting a runtime state in the deployment spec be an option? E.g. trying to see if that's just a missing feature from the script or something else.

**Comment by hijakk** [comment]
Definitely, that sounds fine too. Command line could be nice as an extra
sanity check for folks when deploying a template too, but we can work with
it whatever way makes sense for implementation for you.
